packages/proxy/src/proxy.ts

```diff
@@ -172,19 +172,22 @@ export async function proxyV1({
   const orgName = proxyHeaders[ORG_NAME_HEADER];
   const endpointName = proxyHeaders[ENDPOINT_NAME_HEADER];
 
-  const cacheKey =
-    "aiproxy/proxy/v1:" +
-    (await digest(
+  // Data key is computed from the input data and used for both the cache key and as an input to the encryption key.
+  const dataKey = (await digest(
       JSON.stringify({
         url,
         body,
         authToken: cacheKeyOptions.excludeAuthToken || authToken,
         orgName: cacheKeyOptions.excludeOrgName || orgName,
         endpointName,
-      }),
+      })
```

Comment:
Did you run the pre-commit hooks? You need to install https://pre-commit.com/ and then just run

```
pre-commit 
```

Reply:
I didn't see a format or prettier command, but ran pre-commit now.